src/Inflector/Inflector.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Inflector;
+
+use Doctrine\Common\Inflector\Inflector as LegacyInflector;
+use Symfony\Component\Inflector\Inflector as SymfonyInflector;
+
+/**
+ * @internal
+ *
+ * @deprecated removed in version 3.0
+ */
+final class Inflector
+{
+    public static function tableize(string $word): string
+    {
+        if (\function_exists('\Symfony\Component\String\u')) {
+            return (string) \Symfony\Component\String\u($word)->snake();
+        }
```

Comment:
Can you use 

```
use function Symfony\Component\String\u;
```

Instead? Also it's possible to use the `class_exists(UnicodeString)` instead of `function_exists` (see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/String/Resources/functions.php#L14-L17)